javascript/javascript2/week1/main.css

```diff
@@ -0,0 +1,121 @@
+body {
+    font-family: "Open Sans", sans-serif;
+    background-color: #F9FBFD;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body, h1, h2 {
+    margin: 0;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    background-color: white;
+    box-shadow: 0 1px 2px rgba(190, 190, 190, 0.5);
+}
+
+header .wrapper {
+    width: 800px;
+    margin: 0 auto;
+    height: 80px;
+    background: #FFFFFF;
+    position: relative;
+}
+
+header .wrapper h1, header .wrapper h2 {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+}
+
+header .wrapper h2 {
+    left: 125px;
+}
+
+/*#FF813F*/
+
+section.table-view-control, section.products, section.cart {
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+section.table-view-control, section.products {
+    margin: 24px auto 12px;
+    background-color: white;
+    box-shadow: 0 1px 2px rgba(190, 190, 190, 0.5);
+    border-radius: 8px;
+}
+
+section.table-view-control {
+    padding: 12px;
+}
+
+section.table-view-control>div, section.table-view-control .filters>div {
+    margin-bottom: 24px;
+}
+
+section.products>ul {
+    padding: 0;
+}
+
+section.products>ul>li>ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 12px 12px 12px 12px;
```

Comment:
You could simplify this into `padding: 12px`

Here is a good overview on ways to simplify padding (or margin): https://www.w3schools.com/css/css_padding.asp